Containerfile

```diff
@@ -22,5 +22,12 @@ FROM registry.access.redhat.com/ubi9/ubi-micro
 COPY --from=ui-build /app/apps/standalone/dist /app/proxy/dist
 COPY --from=proxy-build /app/flightctl-ui /app/proxy
 WORKDIR /app/proxy
+LABEL \
+  com.redhat.component="flightctl-ui-container" \
+  description="Flightctl User Interface Service" \
```

Comment:
```suggestion
  description="Flight Control User Interface Service" \
```

Reply:
Based on the discussion on the Backend [PR](https://github.com/flightctl/flightctl/pull/1166), the naming in this PR is correct.